protocol/communities_messenger_test.go

```diff
@@ -3684,3 +3685,254 @@ func (s *MessengerCommunitiesSuite) TestRetrieveBigCommunity() {
+	s.Require().Equal(notification.Dismissed, false)
+
+	// Pull message and make sure org is received
+	err = tt.RetryWithBackOff(func() error {
+		response, err = s.alice.RetrieveAll()
+		if err != nil {
+			return err
+		}
+		if len(response.Communities()) == 0 {
+			return errors.New("community not received")
+		}
+		return nil
+	})
+
+	s.Require().NoError(err)
+	s.Require().NotNil(response)
+
+	s.Require().Len(response.RequestsToJoinCommunity, 1)
+	s.Require().Equal(communities.RequestToJoinStateAccepted, response.RequestsToJoinCommunity[0].State)
+
+	s.Require().Len(response.ActivityCenterNotifications(), 1)
+
+	notification = response.ActivityCenterNotifications()[0]
+	s.Require().NotNil(notification)
+	s.Require().Equal(notification.Type, ActivityCenterNotificationTypeCommunityRequest)
+	s.Require().Equal(notification.MembershipStatus, ActivityCenterMembershipStatusAccepted)
+	s.Require().Equal(notification.Read, false)
+	s.Require().Equal(notification.Accepted, false)
+	s.Require().Equal(notification.Dismissed, false)
+
+	s.Require().Len(response.Communities(), 1)
+	aliceCommunity := response.Communities()[0]
+
+	s.Require().Equal(community.ID(), aliceCommunity.ID())
+	s.Require().True(aliceCommunity.HasMember(&s.alice.identity.PublicKey))
+
+	// Community should be joined at this point
+	s.Require().True(aliceCommunity.Joined())
+
+	// Make sure the requests are not pending on either sides
+	requestsToJoin, err = s.bob.PendingRequestsToJoinForCommunity(community.ID())
+	s.Require().NoError(err)
+	s.Require().Len(requestsToJoin, 0)
+
+	requestsToJoin, err = s.alice.MyPendingRequestsToJoin()
+	s.Require().NoError(err)
+	s.Require().Len(requestsToJoin, 0)
+
+	// Alice shares community with Bob
+
+	response, err = s.alice.ShareCommunityV2(&requests.ShareCommunity{
+		CommunityID:   community.ID(),
+		Users:         []types.HexBytes{common.PubkeyToHexBytes(&s.bob.identity.PublicKey)},
+		InviteMessage: "Come to status Bob!",
+	})
+
+	s.Require().NoError(err)
+	s.Require().NotNil(response)
+	s.Require().Len(response.Messages(), 1)
+
+	err = tt.RetryWithBackOff(func() error {
+		response, err = s.bob.RetrieveAll()
+		if err != nil {
+			return err
+		}
+
+		if len(response.Messages()) == 0 {
+			return errors.New("Messages not received")
+		}
+
+		return nil
+	})
+
+	s.Require().Len(response.Messages(), 1)
+	s.Require().Contains(response.Messages()[0].Text, "Come to status Bob!\nhttps://status.app/c/GzIAAGS9bQ9sj5IgfISkFKWb9HDM_6A1mUUggqTTeYsdg9o011QUrSPim-1Teh7f25WhxjYA")
+	s.Require().NotNil(response.Messages()[0].UnfurledStatusLinks)
+	s.Require().Equal(response.Messages()[0].UnfurledStatusLinks.GetUnfurledStatusLinks()[0].GetCommunity().MembersCount, uint32(2))
+	s.Require().Equal(response.Messages()[0].UnfurledStatusLinks.GetUnfurledStatusLinks()[0].GetCommunity().DisplayName, "status")
+	s.Require().NotEmpty(response.Messages()[0].UnfurledStatusLinks.GetUnfurledStatusLinks()[0].GetCommunity().Banner)
```

Comment:
Actually this is tested as part of `MessengerLinkPreviewsTestSuite`, so IMO there's no sense to test it here too. It only adds cognitive load. 
The only important thing to check I think is the `...GetCommunity().CommunityID`.